pkg/admin/types.go

```diff
@@ -136,6 +137,16 @@ type zkChangeNotification struct {
 	EntityPath string `json:"entity_path"`
 }
 
+type TopicURPsInfo struct {
+	Name       string          `json:"name"`
+	Partitions []PartitionInfo `json:"partitions"`
+}
+
+type TopicOPsInfo struct {
+	Name       string            `json:"name"`
+	Partitions []kafka.Partition `json:"partitions"`
```

Comment:
I think we should be converting `[]kafka.Partition` into `[]PartitionInfo` for consistency